src/pages/SearchPage.js

```diff
@@ -117,6 +132,14 @@ class SearchPage extends Component {
         return sections;
     }
 
+    searchServerData() {
+        const reportIdsToIgnore = _.chain(this.props.reports)
+            .filter()
+            .map(report => report.reportID)
+            .value();
+        ReportActions.searchReports(reportIdsToIgnore, this.state.searchValue.trim());
```

Comment:
Wrong spot for this method I think and should be something like `Report.searchChats()`

Reply:
Moved